railties/lib/rails/generators/rails/app/templates/config/routes.rb.tt

```diff
@@ -9,6 +9,7 @@ Rails.application.routes.draw do
   # Render dynamic PWA files from app/views/pwa/* (remember to link manifest in application.html.erb)
   # get "manifest" => "rails/pwa#manifest", as: :pwa_manifest
   # get "service-worker" => "rails/pwa#service_worker", as: :pwa_service_worker
+  # get ".well-known/appspecific/com.chrome.devtools.json", to: "rails/devtools#show"
```

Comment:
We shouldn't add framework routes to people's `config/routes`, especially since in this case it's meant to be a development only route.

Look for `initializer :add_internal_routes` in `finisher.rb` to see how to do that cleanly.